Tests/AMPStorageTests.m

```diff
@@ -0,0 +1,144 @@
+//
+//  AMPStorageTests.m
+//
+//
+//  Created by Dante Tam on 7/22/21.
+//  Copyright © 2021 Amplitude. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "AMPStorage.h"
+
+@interface AMPStorageTests : XCTestCase
+    
+@end
+
+@implementation AMPStorageTests {}
+
+NSString *exampleEvent = @"{\n    \"api_properties\" :     {\n        \"ios_idfv\" : \"2D623DDA-4AC8-4A1E-BB4B-8264CB2AB31A\"\n    },\n    \"carrier\" : \"Unknown\",\n    \"country\" : \"United States\",\n    \"device_id\" : \"8C027883-5AA1-43F8-8FB8-7837EBE519C4\",\n    \"event_type\" : \"testEvent\",\n \"user_id\" : \"userId\",\n    \"user_properties\" :     {\n    },\n    \"uuid\" : \"41EEDEBB-3A7E-4EDF-BEFC-689D7912BCF3\"\n}";
+
+- (void)setUp {
+    [super setUp];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    NSString* dir = [AMPStorage getAppStorageAmpDir:@""];
+    NSFileManager *fileMgr = [NSFileManager defaultManager];
+    NSArray *fileArray = [fileMgr contentsOfDirectoryAtPath:dir error:nil];
+    for (NSString *filename in fileArray)  {
+        [fileMgr removeItemAtPath:[dir stringByAppendingPathComponent:filename] error:NULL];
+    }
+}
+
+- (void)testHasFileStorage {
+    BOOL hasFileStorage = [AMPStorage hasFileStorage:@"INSTANCE_NAME"];
```

Comment:
In our code base, the instance name should be always lowercase. Do you think we need to make it consistant?

Reply:
It's just for the test. Instance name here is just to match with the string later